railgun/test/integration/knative_test.go

```diff
@@ -0,0 +1,286 @@
+//+build integration_tests
+
+package integration
+
+import (
+	"crypto/tls"
+	"net"
+	"net/http"
+	"os"
+	"os/exec"
+	"time"
+
+	"bytes"
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	types "k8s.io/apimachinery/pkg/types"
+
+	"github.com/kong/kubernetes-testing-framework/pkg/kind"
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	knativeCrds = "https://github.com/knative/serving/releases/download/v0.13.0/serving-crds.yaml"
+	knativeCore = "https://github.com/knative/serving/releases/download/v0.13.0/serving-core.yaml"
+)
+
+func isKnativeReady(ctx context.Context, cluster kind.Cluster) bool {
+	timeout := 1
+	for timeout < 60 {
+		podList, err := cluster.Client().CoreV1().Pods("knative-serving").List(ctx, metav1.ListOptions{})
+		if err != nil {
+			fmt.Println("failed retrieving knative pods.", err)
```

Comment:
If you make this testing function accept a `testing.T` object you'll be able to use the `t.Log` methods to log these instead of printing directly to stdout, which will be helpful for making sure the logging output from this function is consistent with the rest of the test logging.

Reply:
Good suggestion, t.Log should be applied more. Thanks.